src/cart_items/dto/findAll-cartItem.dto.ts

```diff
@@ -6,3 +6,16 @@ export class FindAllCartItemDto extends PaginationQueryDto {
   @IsOptional()
   user_id: string;
 }
+
+export class FindAllCartItemByAdminDto extends PaginationQueryDto {
+  @IsString()
+  @IsOptional()
+  user_id: string;
+
+  @IsString()
```

Comment:
what are sort and number

Reply:
I am not clear your question and could you explain more?